src/lib/Form/Type/ContentType/SortFieldChoiceType.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\ContentType;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Translation\Exception\InvalidArgumentException;
+
+/**
+ * Form type for sort field selection.
+ */
+class SortFieldChoiceType extends AbstractType
+{
+    /** @var TranslatorInterface */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'choices' => $this->getSortFieldChoices(),
+            'choices_as_values' => true,
+            'translation_domain' => 'ezrepoforms_content_type',
```

Comment:
`ezrepoforms` domain name is reserved for `ezsystems/repository-forms`. You can use `forms` domain name.